moped-editor/src/components/LookupSelectComponent.js

```diff
@@ -1,41 +1,47 @@
 import React from "react";
 import { MenuItem, Select } from "@mui/material";
 import { handleKeyEvent } from "../utils/materialTableHelpers";
+import { useGridApiContext } from "@mui/x-data-grid-pro";
 
 /**
  * Component for dropdown select using a lookup table as options
  * @param {*} props
  * @returns {React component}
  */
-const LookupSelectComponent = (props) => (
-  <Select
-    variant="standard"
-    style={{ minWidth: "8em" }}
-    id={props.name}
-    value={props.value || props.defaultValue}
-  >
-    {props.data.map((item) => (
-      <MenuItem
-        onChange={() => props.onChange(item[`${props.name}_id`])}
-        onClick={() => props.onChange(item[`${props.name}_id`])}
-        onKeyDown={(e) => handleKeyEvent(e)}
-        value={item[`${props.name}_id`]}
-        key={item[`${props.name}_name`]}
-      >
-        {item[`${props.name}_name`]}
-      </MenuItem>
-    ))}
-    {props.columnDef.title === "Program" && (
```

Comment:
i looked for where we need this Program version and I could not find it being used